DOCS-726 - Repos section for docs (#5816)
EnterpriseDB/docs: advocacy_docs/repos/getting_started/with_cli/get_your_token.mdx

---
title: Get your token
navTitle: Get your token
description: How to get the token you need to download EDB software
---

## Tokens and EDB Accounts

To download EDB software, you need a token. 
This token is a string that identifies you and your eligibility to download software from EDB. 
If you have an account on EDB, you have already have a token. 
We'll show you how to retrieve it.

If you don't have an account, [sign up for a free EDB account](https://www.enterprisedb.com/accounts/register). 

> Signing up gives you a trial subscription to EDB's software repositories. 

Reviewer: ```suggestion
Signing up gives you a sixty-day trial subscription to EDB's software repositories. 
```

## Getting your token

To get your token, you need use the web interface to the EDB Repositories. This is the only place where tokens are issued. Once you have your token, you'll be able to return to the CLI and complete your work there. 

1. Go to the [EDB Repos 2.0](https://edbpostgres.com/repos-downloads) page, where you can obtain your Repos 2.0 token. 

>     You may have to sign in to your EDB account to access this page. If you do have to sign in, you'll go to the account profile page. On the account profile page, select **Repo Access** to continue.

Reviewer 2: The first time I tested this, it went exactly the way you described it. 
I tried again now, and instead of landing on the "account profile page", I landed on enterprisedb.com 😢 

Maybe we can generalize a bit?

```suggestion
    You may have to sign in to your EDB account to access this page. If you do have to sign in, go to your account profile page (accessible via **My Account** > **Account Settings**). On the account profile page, select **Repo Access** to continue.
```

2.  You'll now be on the EDB Repositories page. On your first visit to this page, select **Request Access** to generate your Repos 2.0 token. 

3. Copy the token using the **Copy Token** icon, and store it safely.

4. Once you’ve created your EDB account and generated your token, you can return to your CLI and configure your system to use the EDB repositories.

> The next step is to [add the other EDB repositories to your system](adding_other_edb_repositories).

Author: ```suggestion
The next step is to [add the other EDB repositories to your system](adding_edb_repositories).
```
